en/docs/learn/configuring-identity-analytics.md

```diff
@@ -17,14 +17,6 @@ To do this, the developers need to view authentication statistics about the logi
 ## Enable analytics
 
 Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and add the following event listeners to enable analytics in WSO2 Identity Server. 
```

Comment:
Shouldn't this now say 'enable the following event publishers to enable analytics'

Reply:
Isn't this this a bit too long for a heading? 
@gomathyk thoughts?

Reply:
Oops. I didn't mean the heading.

I actually suggested a change in the text in the first sentence after the heading

Reply:
Ah yes. I see what you mean